windows/source/serviceConfig.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8" ?>

```

Comment:
@rmrlangford Why is there another copy of this file?

Reply:
@lauravignoli  visual Studio.Net "packages" the solution files like this.